airbyte-integrations/connectors/source-amazon-ads/integration_tests/spec.json

```diff
@@ -17,14 +23,6 @@
         "airbyte_secret": true,
         "type": "string"
       },
-      "scope": {
```

Comment:
why was scope  removed?

Reply:
According to amazon auth [docs](https://advertising.amazon.com/API/docs/en-us/concepts/authorization/authorization-grants#the-consent-request-url) we need only scope to get `refresh_token`.
and we don't need scope to exchange `refresh_token` -> `access_token`.

It means scope is useless here because we already have `refresh_token` as connector input parameter.

I this PR I moved `scope` from python side to Java side 
because we need scope only on stage where we get `refresh_token`